assets/xml/objects/object_dekubaba.xml

```diff
@@ -1,27 +1,40 @@
-        <Limb Name="object_dekubaba_Standardlimb_0029DC" Type="Standard" EnumName="OBJECT_DEKUBABA_LIMB_02" Offset="0x29DC" />
-        <Limb Name="object_dekubaba_Standardlimb_0029E8" Type="Standard" EnumName="OBJECT_DEKUBABA_LIMB_03" Offset="0x29E8" />
-        <Limb Name="object_dekubaba_Standardlimb_0029F4" Type="Standard" EnumName="OBJECT_DEKUBABA_LIMB_04" Offset="0x29F4" />
-        <Limb Name="object_dekubaba_Standardlimb_002A00" Type="Standard" EnumName="OBJECT_DEKUBABA_LIMB_05" Offset="0x2A00" />
-        <Limb Name="object_dekubaba_Standardlimb_002A0C" Type="Standard" EnumName="OBJECT_DEKUBABA_LIMB_06" Offset="0x2A0C" />
-        <Limb Name="object_dekubaba_Standardlimb_002A18" Type="Standard" EnumName="OBJECT_DEKUBABA_LIMB_07" Offset="0x2A18" />
-        <Skeleton Name="object_dekubaba_Skel_002A40" Type="Normal" LimbType="Standard" LimbNone="OBJECT_DEKUBABA_LIMB_NONE" LimbMax="OBJECT_DEKUBABA_LIMB_MAX" EnumName="ObjectDekubabaLimb" Offset="0x2A40" />
-        <DList Name="object_dekubaba_DL_003070" Offset="0x3070" />
-        <Texture Name="object_dekubaba_Tex_0031C0" OutName="tex_0031C0" Format="rgba16" Width="32" Height="32" Offset="0x31C0" />
+
+        <!-- Deku Baba animations -->
+        <Animation Name="gDekuBabaPauseChompAnim" Offset="0x208"/>
+        <Animation Name="gDekuBabaFastChompAnim" Offset="0x2B8"/>
+
+        <!-- Deku Baba body DLists (not all are limb DLists) -->
+        <DList Name="gDekuBabaLowerJawDL" Offset="0x580"/>
+        <DList Name="gDekuBabaUpperJawDL" Offset="0xB68"/>
+        <DList Name="gDekuBabaBaseLeavesDL" Offset="0x10F0"/>
+        <DList Name="gDekuBabaStemTopDL" Offset="0x1330"/>
+        <DList Name="gDekuBabaStemMiddleDL" Offset="0x1628"/>
+        <DList Name="gDekuBabaStemBaseDL" Offset="0x1828"/>
+
+        <!-- Deku Baba body textures -->
+        <Texture Name="gDekuBabaJawOuterTex" OutName="deku_baba_jaw_outer" Format="rgba16" Width="16" Height="32" Offset="0x18D0"/>
+        <Texture Name="gDekuBabaJawInnerTex" OutName="deku_baba_jaw_inner" Format="rgba16" Width="32" Height="32" Offset="0x1CD0"/>
+        <Texture Name="gDekuBabaLeafTex" OutName="deku_baba_leaf" Format="rgba16" Width="16" Height="32" Offset="0x24D0"/>
+        <Texture Name="gDekuBabaBulbTex" OutName="deku_baba_bulb" Format="rgba16" Width="8" Height="8" Offset="0x28D0"/>
+        <Texture Name="gDekuBabaStemTex" OutName="deku_baba_stem" Format="rgba16" Width="8" Height="8" Offset="0x2950"/>
+
+        <!-- Deku Baba limbs (upper/lower is only based on the "tongue" being in the lower) -->
+        <Limb Name="gDekuBabaRootLimb" Type="Standard" EnumName="DEKUBABA_LIMB_ROOT" Offset="0x29D0" />
+        <Limb Name="gDekuBabaUpperJawRootRootLimb" Type="Standard" EnumName="DEKUBABA_LIMB_UPPER_JAW_ROOT_ROOT" Offset="0x29DC" />
+        <Limb Name="gDekuBabaUpperJawRootLimb" Type="Standard" EnumName="DEKUBABA_LIMB_UPPER_JAW_ROOT" Offset="0x29E8" />
```

Comment:
Should this use the `Root` and `Wrapper` terminology instead?

Reply:
```
Normal skeleton at offset 2A40, 7 standard limbs
limbTable offset: 2A24
(    0,    0,    0),    1,     -1, 0x00000000  0
(    0,   43, -114),    2,      4, 0x00000000  └─ 1
(    0,    0,    0),    3,     -1, 0x00000000     ├─ 2
(    0,    0,    0),   -1,     -1, 0x06000B68     │  └─ 3*
(    0, -155, -144),    5,     -1, 0x00000000     4
(    0,    0,    0),    6,     -1, 0x00000000     └─ 5
(    0,    0,    0),   -1,     -1, 0x06000580        └─ 6*
```
Possibly?

Reply:
Looks like it. limbs 1 and 4 would be the root, while 2 and 5 would be the wrapper (since they are at offset 0,0,0)